helm-chart/binderhub/values.yaml

```diff
@@ -133,6 +133,7 @@ deployment:
 
 dind:
   enabled: false
+  initContainers:
```

Comment:
Can we default this to an array ([]) so it is clear it's an array type we expect here?